Packs/ProofpointServerProtection/Integrations/ProofpointProtectionServerV2/ProofpointProtectionServerV2_description.md

```diff
@@ -0,0 +1,14 @@
+### Authentication
+An administrator must have a role that includes access to a specific REST API. 
+
+Proofpoint on Demand (PoD) administrators must file a support ticket to Proofpoint support to obtain a role with access to an API.
+
+On premise administrators: Edit the **filter.cfg** file and set the following key to true: com.proofpoint.admin.apigui.enable=t
```

Comment:
`=t`? or are you missing the rest of the word?

Reply:
`=t`